tensorrt-onnx-graphsurgeon/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.2.0" %}                                                              #[ppc64le]
+{% set sha256 = "03ce0427686e19d60919f6ba68dea5446a26ac53f4398bdea4ec96aec30740e7" %}    #[ppc64le]
+{% set trt_tar = "TensorRT-7.2.0.13.CentOS-7.6.ppc64le-gnu.cuda-11.0.cudnn8.0.tar.gz" %} #[ppc64le]
+
+{% set version = "0.2.6" %}                                                             #[x86_64]
+{% set sha256 = "3d42890424e2a2a155df67aaace45d001a95f12d6781008c5dcf62ea480ea664" %}   #[x86_64]
+{% set trt_tar = "TensorRT-7.2.1.6.CentOS-7.6.x86_64-gnu.cuda-11.0.cudnn8.0.tar.gz" %}  #[x86_64]
+
+{% set trt_tar_dir =  environ.get('LOCAL_SRC_DIR', 0) %}
+
+package:
+  name: onnx-graphsurgeon
+  version: {{ version }} 
+
+source:
+  - url: file://{{ trt_tar_dir }}/{{ trt_tar }}
+    sha256: {{ sha256 }}
+    folder: tensorrt
+
+build:
+  number: 1
+  string: py{{ python | replace(".", "") }}_{{ PKG_BUILDNUM }}
+  
+requirements:
+  host:
+    - python {{ python }}
+    - wheel {{ wheel }}
+  run:
+    - python {{ python }}
+
+about:
+  home: https://developer.nvidia.com/tensorrt
+  license: NVIDIA Software License
+  summary: graphsurgeon allows you to transform TensorFlow graphs.
+  description: |
+    graphsurgeon allows you to transform TensorFlow graphs. Its capabilities are broadly divided into two categories - search and manipulation. 
+    Search functions allow you to find nodes in a TensorFlow graph. Manipulation functions allow you to modify, add, or remove nodes.
+  doc_url: https://docs.nvidia.com/deeplearning/sdk/tensorrt-api/python_api/graphsurgeon/graphsurgeon.html
```

Comment:
I see the same information in about section of graphsurgeon too. Hope this is fine.

Reply:
yeah- there are no online docs for `onnx-graphsurgen` yet, so I put in the details for `graphsurgeon`